pyproject.toml

```diff
@@ -48,16 +48,16 @@ classifiers = [
 
 [tool.poetry.dependencies]
 python = ">=3.7,<3.12"
-boto3 = ">=1.3.0,<2"
+boto3 = "^1.3.0"
 click = ">=7.0,<9.0"
 cfn-flip = "^1.2.3"
 deepdiff = "^5.5"
 deprecation = "^2.0"
 jinja2 = "^3.0"
-jsonschema = ">=3.2,<3.3"
-networkx = ">=2.6,<2.7"
+jsonschema = "~3.2"
```

Comment:
[Tilde notation](https://python-poetry.org/docs/dependency-specification/#tilde-requirements) literally means ">={this version},<{next minor version}". Again, no actual change here, just improving the notation.